app/generators/cinema_hall.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Generators
+  class CinemaHall
+    AVAILABLE_COLUMNS = ('A'..'J')
+
+    def generate_seats(capacity)
+      seats = []
+      (1..capacity / AVAILABLE_COLUMNS.count).each do |row|
+        AVAILABLE_COLUMNS.each { |column| seats.push("#{row}#{column}") }
+      end
+      seats
+    end
```

Comment:
Hm, for capacity 10 it would return 10 seats, but for 15 it would return also 10... are you ok with that? :)

Plus a little more idiomatic ruby:

```ruby
def generate_seats(capacity)
  (1..capacity / AVAILABLE_COLUMNS.size).to_a.product(AVAILABLE_COLUMNS.to_a).map(&:join)
end
```